tasks/install.yml

```diff
@@ -4,14 +4,29 @@
     gather_subset:
       - distribution
 
+- name: Enable EPEL
+  dnf:
+    name: epel-release
```

Comment:
I am worried this is going to cause other problems, but lets not change this for now, I get it has just moved about the the file.

Reply:
Ceph install instructions say it requires EPEL: https://docs.ceph.com/en/latest/install/get-packages/#id3